deploy/compliance/aquasecurity.github.io_clustercompliancedetailreports.yaml

```diff
@@ -2,40 +2,40 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: clustercompliancedetailreports.aquasecurity.github.io
   labels:
     app.kubernetes.io/managed-by: trivy-operator
-    app.kubernetes.io/version: "0.1.0"
+    app.kubernetes.io/version: 0.1.0
```

Comment:
I think we should keep the quotes here. Yaml has a tendency to misinterpret values like this.... Maybe VSCode is not the right tool to use? Maybe try `yq` or `dyff` - to avoid changes except the wanted (sort keys)?

Same comment on the other files.

Reply:
Removing quotes is equivalent to the quoted strings. Even the other tools `yq` and `dyff` produce unquoted strings.

Reply:
It seems like most tools to sort Yaml will remove the quotes. And that makes sense, since `0.1.0` is equivalent to `"0.1.0"` in Yaml. But if we get versions like `1.1`, this will become problematic, but highly unlikely - since I presume we use SemVer here. I am ok with removing the quotes in this PR.